src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.cs

```diff
@@ -0,0 +1,227 @@
+                             DateTime.Now
+                             );
+                            var updatedTime = await CheckForError(previousTransactionDateTime, DependencyUpdateErrorProcessorOptions.GithubUrl);
+                            await update.TryUpdateAsync(
+                                tx,
+                                "update",
+                                updatedTime.UtcDateTime,
+                                previousTransactionDateTime
+                                );
+                            await tx.CommitAsync();
+                        }
+                    }
+                    catch (Exception ex)
+                    {
+                        Logger.LogError(ex, "Processing update dependency messages");
+                    }
+                }
+            }
+            else
+            {
+                Logger.LogInformation("Dependency Update Error processor is disabled because no App Configuration was available.");
+            }
+        }
+
+        public async Task<DateTimeOffset> CheckForError(DateTimeOffset previousTransactionDateTime , string whereToCreateIssue)
+        {
+            List<RepositoryBranchUpdateHistoryEntry> list = (from repo in Context.RepositoryBranchUpdateHistory
+                                                             where repo.Success == false
+                                                             where repo.Timestamp > previousTransactionDateTime.UtcDateTime
+                                                             orderby repo.Timestamp ascending
+                                                             select repo).ToList();
+            if (!list.Any())
+            {
+                return previousTransactionDateTime;
+            }
+
+            Logger.LogInformation($"Going to create the github issue");
+            try
+            {
+                await CreateGitHubIssueAsync(previousTransactionDateTime.UtcDateTime,  list[0].Repository, list[0].Branch, 
+                    list[0].Action, list[0].ErrorMessage, list[0].Method , whereToCreateIssue);
+            }
+            catch (Exception ex)
+            {
+                Logger.LogError(ex, "Could not create a github issue for the error: " + list[0].ErrorMessage);
+            }
+            previousTransactionDateTime = list[0].Timestamp;
+            return previousTransactionDateTime;
+        }
+
+        public async Task<bool> SearchCreatedIssueAsync(DateTime transactionDateTime , string errorMessage)
+        {
+            var query = new StringBuilder();
+            HttpClient client = new HttpClient
+            {
+                BaseAddress = new Uri("https://api.github.com")
+            };
+            client.DefaultRequestHeaders.Add("User-Agent", @"Mozilla/5.0 (Windows NT 10; Win64; x64; rv:60.0) Gecko/20100101 Firefox/60.0");
+
+            query.Append($"is:issue+is:open+{"Error Message: " + errorMessage}in:title&order=desc");
```

Comment:
Why do we have a template string that escapes into code... but then concatenates the string???

This should jsut be 
```suggestion
            query.Append($"is:issue+is:open+Error Message: {errorMessage}in:title&order=desc");
```

I think it's also not right at that point.  it's going to see "error message: thing" as some "message: " keyword. Also, the message needs to be quoted. Also, there is no space after the "errorMessage" so if the message is "pizza", this is going to be `pizzain:title` which doesn't mean anything. Also, spaces are not valid URL characters. Also, we don't actually put "Error Message" in the title, only "Error", so even if this query worked, it still wouldn't find them...